src/components/navProfile/Avatar.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import AvatarEdit from 'react-avatar-edit';
+import Modal from 'react-modal';
```

Comment:
why do you install additional lib???
sematic-ui has its own Modal component
https://react.semantic-ui.com/modules/modal/

Reply:
okay